iot_config/src/org_service.rs

```diff
@@ -158,11 +170,83 @@ impl iot_config::Org for OrgService {
         }))
     }
 
-    async fn disable(&self, _request: Request<OrgDisableReqV1>) -> GrpcResult<OrgDisableResV1> {
-        unimplemented!()
+    async fn disable(&self, request: Request<OrgDisableReqV1>) -> GrpcResult<OrgDisableResV1> {
+        let request = request.into_inner();
+
+        let req = self.verify_admin_signature(request)?;
+
+        if org::get_status(req.oui, &self.pool)
+            .await
+            .map_err(|_| Status::internal("error retrieving current status"))?
+            == OrgStatus::Enabled
+        {
+            org::toggle_status(req.oui, OrgStatus::Disabled, &self.pool)
+                .await
+                .map_err(|_| Status::internal(format!("org disable failed for: {}", req.oui)))?;
+
+            let org_routes = list_routes(req.oui, &self.pool).await.map_err(|_| {
+                Status::internal(format!(
+                    "error retrieving routes for disabled org: {}",
+                    req.oui
+                ))
+            })?;
+
+            for route in org_routes {
```

Comment:
Does it matter if the org is successfully disabled but one or more deletes of the routes fail ?  what are the implications of that ?
Same question applies to the org enable flow

Reply:
My understanding is that `send` fails when there are no receivers. 
If there are no receivers, we probably don't need to continue trying to send updates. 
But we should not error out to the caller.

Reply:
sgtm,  I read the docs but wasnt entirely sure offline queuing or delivery guarantees were supported